runtime/jextractnatives/jextractnatives.c

```diff
@@ -127,70 +120,76 @@ validateDump(JNIEnv *env)
 			return JNI_FALSE;
 		}
 
 		if (eyecatcher == (jlong)-1) {
-			j9str_printf(PORTLIB, 
-					errBuf, sizeof(errBuf), 
+			j9str_printf(PORTLIB,
+					errBuf, sizeof(errBuf),
 					"JVM anchor block (J9VMRAS) not found in dump. Dump may be truncated, corrupted or contains a partially initialized JVM.");
 			(*env)->ThrowNew(env, errorClazz, errBuf);
 			return JNI_FALSE;
 		}
 
-#if !defined (J9VM_ENV_DATA64)
-		if ( (U_64)eyecatcher > (U_64)0xFFFFFFFF ) {
-			j9str_printf(PORTLIB, 
-				errBuf, sizeof(errBuf), 
-				"J9RAS is out of range for a 32-bit pointer (0x%16.16llx). This version of jextract is incompatible with this dump.", 
+#if !defined(J9VM_ENV_DATA64)
+		if ((U_64)eyecatcher > (U_64)0xFFFFFFFF) {
+			j9str_printf(PORTLIB,
+				errBuf, sizeof(errBuf),
+				"J9RAS is out of range for a 32-bit pointer (0x%16.16llx). This version of jextract is incompatible with this dump.",
 				eyecatcher);
 			(*env)->ThrowNew(env, errorClazz, errBuf);
 			return JNI_FALSE;
 		}
 #endif
-		/* Allocate this, since on 64bit platforms we want to know now that we can't allocate
-		 * the scratch space. This allows us to exit early with a simple error message */
+		/* Allocate this, since on 64-bit platforms we want to know now that we can't allocate
+		 * the scratch space. This allows us to exit early with a simple error message.
+		 */
 		ras = dbgMallocAndRead(sizeof(J9RAS), (void *)(UDATA)eyecatcher);
 		if (ras != NULL) {
 			if (ras->bitpattern1 == 0xaa55aa55 && ras->bitpattern2 == 0xaa55aa55) {
 				if (ras->version != J9RASVersion) {
-					j9str_printf(PORTLIB, 
-						errBuf, sizeof(errBuf), 
-						"J9RAS.version is incorrect (found %u, expecting %u). This version of jextract is incompatible with this dump.", 
+					j9str_printf(PORTLIB,
+						errBuf, sizeof(errBuf),
+						"J9RAS.version is incorrect (found %u, expecting %u). This version of jextract is incompatible with this dump.",
 						ras->version,
 						J9RASVersion);
 					(*env)->ThrowNew(env, errorClazz, errBuf);
 					return JNI_FALSE;
 				}
 				if (ras->length != sizeof(J9RAS)) {
-					j9str_printf(PORTLIB, 
-						errBuf, sizeof(errBuf), 
-						"J9RAS.length is incorrect (found %u, expecting %u). This version of jextract is incompatible with this dump.", 
+					j9str_printf(PORTLIB,
+						errBuf, sizeof(errBuf),
+						"J9RAS.length is incorrect (found %u, expecting %u). This version of jextract is incompatible with this dump.",
 						ras->length,
 						sizeof(J9RAS));
 					(*env)->ThrowNew(env, errorClazz, errBuf);
 					return JNI_FALSE;
 				}
 				if (ras->buildID != J9UniqueBuildID) {
-					j9str_printf(PORTLIB, 
-						errBuf, sizeof(errBuf), 
-						"J9RAS.buildID is incorrect (found %llx, expecting %llx). This version of jextract is incompatible with this dump.", 
+					j9str_printf(PORTLIB,
+						errBuf, sizeof(errBuf),
+						"J9RAS.buildID is incorrect (found %llx, expecting %llx). This version of jextract is incompatible with this dump.",
```

Comment:
I think the option to disable the check should be mentioned in this message.

Reply:
I thought of that, but I wasn't sure how obscure the bypass should be. If we're going to explicitly call it out, perhaps it should just be a normal command-line option?

Reply:
I'm not sure how big the impact to service will be. If it turns out some customers are running jextract from a different JVM version, we should mitigate the impact by making it easy to find the workaround. I don't have strong feelings either way for sticking with the current option or making it a command line option.

Reply:
It's both easier to talk about and use if it's a command-line option: how about `-r` (relax checking)?
I'll also document the remaining supported options.

Reply:
Seems consistent with the other options in the help, -f and -v.